spec/lib/miq_automation_engine/service_methods/miq_ae_service_classification_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+include AutomationSpecHelper
+
+describe MiqAeMethodService::MiqAeServiceClassification do
+  before do
+    @cat1 = FactoryGirl.create(:classification_department_with_tags)
+    @cat2 = FactoryGirl.create(:classification_cost_center_with_tags)
+    @cat_array = Classification.categories.collect(&:name)
+    @tags_array = @cat2.entries.collect(&:name)
+  end
+
+  let(:user) { FactoryGirl.create(:user_with_group) }
+
+  def setup_model(method_script)
+    create_ae_model_with_method(:method_script => method_script,
+                                :name          => 'VITALSTATISTIX',
+                                :ae_namespace  => 'OBELIX',
+                                :ae_class      => 'ASTERIX',
+                                :instance_name => 'DOGMATIX',
+                                :method_name   => 'GETAFIX')
+  end
+
+  def invoke_ae
+    MiqAeEngine.instantiate("/OBELIX/ASTERIX/DOGMATIX", user)
+  end
+
+  it "get a list of categories" do
+    setup_model("$evm.root['result'] = $evm.vmdb('Classification').categories")
```

Comment:
@mkanoor What do you think about testing this by directly instantiating the Classification service model and calling the methods on that object like we do in [miq_ae_service_user_spec.rb](https://github.com/ManageIQ/manageiq/blob/master/spec/lib/miq_automation_engine/service_methods/miq_ae_service_user_spec.rb#L12) instead of setting up the framework to run an automate method?

The only added benefit I see from running the automate method is testing that `$evm.vmdb('<model_name>')` is working but we can test that directly and not have to test it here.  I would like to see this type of change for more of the service model specs.

Thoughts?